src/main/java/duke/action/Todo.java

```diff
@@ -0,0 +1,54 @@
+package duke.action;
+
+/**
+ * An extension from Action.
+ */
+public class Todo  extends Action {
+
+    /**
+     * Constructs a new Todo object with a String variable s.
+     * @param s task todo
+     */
+    public Todo(String s)  {
```

Comment:
```suggestion
    public Todo(String s) {
```
Coding standard spacing